.gitignore

```diff
@@ -1,2 +1,5 @@
 __pycache__/
 app.db
+
+virtualenv
+venv
```

Comment:
guess just one of those is needed, i would propose to go with the hidden file ".virtualenv" instead